squad/frontend/ci.py

```diff
@@ -16,7 +16,7 @@ def testjobs(request, group_slug, project_slug, build_version):
         raise Http404()
     testjobs = TestJob.objects.filter(
         target=project,
-        build=build.version
+        target_build=build.id
```

Comment:
this will work due because the Django ORM will take this, but it's inconsistent with all other similar changes. use just  `build` insead of `build.id` (unless I am missing something)